src/main/java/org/takes/facets/fork/FkContentType.java

```diff
@@ -49,25 +50,25 @@
     private final transient MediaTypes type;
 
     /**
-     * Response to return.
+     * Take to handle the request and dynamically return the response.
      */
-    private final transient Response origin;
+    private final transient Take take;
 
     /**
      * Ctor.
      * @param atype Accepted type
-     * @param response Response to return
+     * @param take Take to handle the request dynamically.
      */
-    public FkContentType(final String atype, final Response response) {
+    public FkContentType(final String atype, final Take take) {
```

Comment:
@dalifreire @HDouss are you guys sure that we should modify this constructor? It's in public API used in many projects, after update they won't compile anymore